projects/core/src/kyma/services/index.ts

```diff
@@ -1,3 +1 @@
-import { OpenIdAuthenticationTokenService } from './open-id-token/open-id-token.service';
-
-export const KymaServices: any[] = [OpenIdAuthenticationTokenService];
```

Comment:
This one is actually a breaking change. You want to proceed with it?

Reply:
thanks, added required migrations info